docker-compose.yml

```diff
@@ -9,7 +9,7 @@ networks:
     #
     # If rabbit is not local then the network is not defined externally and we
     # need to define it ourselves (set this to false)
-    external: ${RABBITMQ_IS_LOCAL:-false}
+#    external: ${RABBITMQ_IS_LOCAL:-false}
```

Comment:
Did you have trouble with this?  I think I set to true in base env and it worked fine.

Reply:
I did, but I probably have it set wrong in my .env. I don't think we ever expect rabbit to be external, so maybe I'll just remove it.